deployment/kubernetes/aws-eks/tf-cluster/060_eks_workernodes_scaling_group.tf

```diff
@@ -86,7 +86,22 @@ resource "aws_autoscaling_group" "eks-autoscaling-group" {
     propagate_at_launch = true
   }
 }
-
+resource "aws_autoscaling_schedule" "scale-down" {
+  scheduled_action_name = "scale-down"
+  min_size = 0
+  max_size = 0
+  desired_capacity = 0
+  recurrence = "00 23 * * *" #Mon-Sun at 12AM CET
```

Comment:
I think, makes sense to add comment that

"You can specify a recurrence schedule, in UTC, using the Unix cron syntax format"

if anyone in the future would like to adjust schedule